src/psyclone/psyGen.py

```diff
@@ -719,7 +719,8 @@ def __init__(self, name, KernFactory, BuiltInFactory, alg_calls=None,
         # Populate the Schedule Symbol Table with the reserved names.
         if reserved_names:
             for reserved in reserved_names:
-                self.symbol_table.add(Symbol(reserved))
+                self.symbol_table.add(Symbol(reserved),
```

Comment:
Is this change related to the fact that you removed one of the entries in reserved names in a file I reviewed earlier?

Reply:
I am not sure, I could revert this change without tests failing.